src/ophys_etl/modules/suite2p_registration/__main__.py

```diff
@@ -71,11 +85,13 @@ def run(self):
                     self.args['smooth_sigma_time_steps'])
 
                 optimize_result = optimize_motion_parameters(
-                    intial_frames,
-                    smooth_sigmas,
-                    smooth_sigma_times,
-                    suite2p_args,
-                    self.logger.info)
+                    initial_frames=intial_frames,
+                    smooth_sigmas=smooth_sigmas,
+                    smooth_sigma_times=smooth_sigma_times,
+                    suite2p_args=suite2p_args,
+                    trim_frames_start=self.args['trim_frames_start'],
```

Comment:
if `initial_frames` already were trimmed, why do you need to pass `trim_frames_start` and `trim_frames_end` here?

Reply:
This method loads in batches from the original file and thus needs to know which frames to trim off.